src/components/SplitVertical/SplitVertical.jsx

```diff
@@ -0,0 +1,392 @@
+import _ from 'lodash';
+import React from 'react';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createClass, filterTypes, omitProps } from '../../util/component-types';
+import DragCaptureZone from '../DragCaptureZone/DragCaptureZone';
+
+const cx = lucidClassNames.bind('&-SplitVertical');
+
+const {
+	any,
+	bool,
+	func,
+	node,
+	number,
+	string,
+	oneOfType,
+} = React.PropTypes;
+
+/**
+ * {"categories": ["helpers"]}
+ *
+ * `SplitVertical` renders a vertical split.
+ */
+const SplitVertical = createClass({
+	displayName: 'SplitVertical',
+
+	propTypes: {
+		/**
+		 * Appended to the component-specific class names set on the root
+		 * element. Value is run through the `classnames` library.
+		 */
+		className: any,
+		/**
+		 * Direct children must be types {Splitvertical.Leftpane, Splitvertical.Divider, Splitvertical.Leftpane}.
+		 * All content is composed as children of these respective elements.
+		 */
+		children: node,
+		/**
+		 * Render as expanded or collapsed.
+		 */
+		isExpanded: bool,
+		/**
+		 * Allows animated expand and collapse behavior.
+		 */
+		isAnimated: bool,
+		/**
+		 * Called when the user resizes the split with the Divider.
+		 *
+		 * Signature: `(width, { event, props }) => {}`
+		 */
+		onResize: func,
+		/**
+		 * Use this prop to shift the collapsed position by a known value.
+		 */
+		collapseShift: number,
+	},
+
+	components: {
+		/**
+		 * Left pane of the split.
+		 */
+		LeftPane: createClass({
+			displayName: 'SplitVertical.LeftPane',
+			propTypes: {
+				/**
+				 * Any valid React children.
+				 */
+				children: node,
+				/**
+				 * Set width of this pane.
+				 */
+				width: oneOfType([number, string]),
+				/**
+				 * Define this pane as the primary content pane. When the split is collapsed, this pane becomes full width.
+				 */
+				isPrimary: bool,
+			},
+			getDefaultProps(){
+				return {
+					isPrimary: false,
+				};
+			},
+		}),
+
+		/**
+		 * Right pane of the split.
+		 */
+		RightPane: createClass({
+			displayName: 'SplitVertical.RightPane',
+			propTypes: {
+				/**
+				 * Any valid React children.
+				 */
+				children: node,
+				/**
+				 * Set width of this pane.
+				 */
+				width: oneOfType([number, string]),
+				/**
+				 * Define this pane as the primary content pane. When the split is collapsed, this pane becomes full width.
+				 */
+				isPrimary: bool,
+			},
+			getDefaultProps(){
+				return {
+					isPrimary: false,
+				};
+			},
+		}),
+
+		/**
+		 * The area that separates the split panes. Can be dragged to resize them.
+		 */
+		Divider: createClass({
+			displayName: 'SplitVertical.Divider',
+			propTypes: {
+				/**
+				 * Any valid React children.
+				 */
+				children: node,
+			},
+		}),
+	},
+
+	getDefaultProps() {
+		return {
+			isExpanded: true,
+			isAnimated: false,
+			collapseShift: 0,
+			onResize: _.noop,
+		};
+	},
+
+	getInitialState() {
+		return {
+			isAnimated: false,
```

Comment:
having `isAnimated` in state is to make sure it doesn't animation on first render, so it's always `false` on first render.